ivy_tests/test_ivy/test_frontends/test_tensorflow/test_random.py

```diff
@@ -118,3 +118,56 @@ def test_tensorflow_shuffle(
         value=values[0],
         seed=seed,
     )
+
+
+# random_stateless_uniform
+@handle_frontend_test(
+    fn_tree="tensorflow.random.stateless_uniform",
+    shape=helpers.dtype_and_values(
+        available_dtypes=("int64", "int32"),
+        min_value=1,
+        max_value=5,
+        min_num_dims=1,
+        max_num_dims=1,
+    ),
+    seed=helpers.dtype_and_values(
+        available_dtypes=("int64", "int32"), min_value=0, max_value=10, shape=[2]
+    ),
+    minval=helpers.ints(min_value=0, max_value=3),
+    maxval=helpers.ints(min_value=4, max_value=10),
+    dtype=helpers.dtype_and_values(
```

Comment:
sorry i guess i need more guidance here because `helpers.get_dtypes()` doesnt accept `available_dtypes=("int32", "int64", "float16", "float32", "float64"),` . I can only choose from ` integer, float, valid, numeric, and unsigned` . So i would have to exclude a lot of cases in the test function

Reply:
Digging deeper into `dtype_and_values` found out that it uses `helpers.array_dtypes()` instead of `helpers.get_dtypes()`. My bad 😅